pyproject.toml

```diff
@@ -26,6 +26,7 @@ packages = [
     { include = "beets" },
     { include = "beetsplug" },
 ]
+include = ["test", "man/**/*"] # extra files to include in the sdist
```

Comment:
what does ** in a path mean again?

Reply:
I tried building using simply `man`, but poetry wouldn't include the files inside this directory. `man/**/*` did the trick though.

`man/**/*` translates to _all files inside the `man` directory, recursively_. See below

![image](https://github.com/user-attachments/assets/c201b477-a19f-4cc1-ac98-f36f9bc00d9f)